localization/english/activities/expeditions/expedition_activity_l_english.yml

```diff
@@ -1,24 +1,24 @@
 ﻿l_english:
   #expedition
   activity_expedition:0 "Expedition"
-  expedition_phase_desc:0 "Explore the ruins, discover what you can"
+  expedition_phase_desc:0 "Explore the ruins, discover what you can."
   activity_expedition_name:0 "[ACTIVITY.GetOwner.GetNamePossessiveOrMy|U] expedition"
-  activity_expedition_desc:4 "Expeditions are about rediscovering the secrets of ancient Elven peoples.\n\nYou and an entourage of knights and adventurers will explore Elven ruins in seek of lost knowledge and treasures"
+  activity_expedition_desc:4 "Expeditions are about rediscovering the secrets of the ancient Elven peoples.\n\nYou and an entourage of knights and adventurers will explore Elven ruins in search of long lost knowledge and great treasure."
```

Comment:
maybe "great treasures" ?

Reply:
Either one works, but treasures does make more sense considering there's a few different rewards you can get